### PR TITLE
chore: one GitHub tag and release per version rather than one per crate

### PR DESCRIPTION
**Pull request by danking**

### CODE REVIEW

**Comment by danking** [comment]
Based on: https://release-plz.ieni.dev/docs/extra/single-tag

**Review by robert3005 — APPROVED** [review]

File: release-plz.toml [hunk 1]
```diff
@@ -18,3 +20,9 @@ body = """
 {% endfor -%}
 {% endfor -%}
 """
+
+[[package]]
```
Comment (a10y) [inline release-plz.toml:24]: why this?

Comment (robert3005) [reply release-plz.toml:24]: there has to be one package that gets tagged per the docs linked

File: release-plz.toml [hunk 2]
```diff
@@ -18,3 +20,9 @@ body = """
...
+name = "vortex-array"
+git_tag_enable = true
+git_release_enable = true
+git_tag_name = "v{{ version }}"
```
Comment (a10y) [inline release-plz.toml]: can i say something without people getting mad

image

Comment (robert3005) [reply release-plz.toml]: Nice, I missed this

File: release-plz.toml (hunk 1 above)
Comment (a10y) [reply release-plz.toml:24]: yea we talked about this IRL, forgot to resolve comment

**Pull request merged**